images/external.md

```diff
@@ -18,6 +18,29 @@ such as [`jupyter/r-notebook`](https://jupyter-docker-stacks.readthedocs.io/en/l
 and [`jupyter/datascience-notebook`](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html#jupyter-datascience-notebook),
 so you can immediately run R on Jupyter.
 
+### [JupyterLab R docker stack](https://github.com/b-data/jupyterlab-r-docker-stack)
```

Comment:
Can you change the header to the repository name as it is confusing with the Jupyter Docker Stacks?

```suggestion
### [b-data/jupyterlab-r-docker-stack](https://github.com/b-data/jupyterlab-r-docker-stack)
```

Reply:
👍